### DISCUSSION TITLE
Nuclei automation for finding bugs

### DISCUSSION THREAD

**Discussion by sravanee98** [projectdiscovery/nuclei-templates#8340] [General]
Does nuclei provide the chain of tools for recon and low hanging bugs if so how can we create custom template and does the scan work for PAAS platform

**Comment by princechaddha** [comment]
Nuclei is a YAML-based, fast, and customizable vulnerability scanner. You can create your own nuclei-templates by reviewing the documentation [here](http://nuclei.projectdiscovery.io/). For reconnaissance and scanning, we offer a variety of tools that can be integrated into your workflow, which can be found [here](https://github.com/projectdiscovery). To understand how you can chain our tools, you can explore these [blog posts](https://blog.projectdiscovery.io/tag/reconnaissance/).
